src/Provider/FileProvider.php

```diff
@@ -330,7 +330,7 @@ public function validate(ErrorElement $errorElement, MediaInterface $media)
         if ('' !== $media->getBinaryContent()->getFilename() && !\in_array($media->getBinaryContent()->getMimeType(), $this->allowedMimeTypes, true)) {
             $errorElement
                 ->with('binaryContent')
-                    ->addViolation('Invalid mime type : %type%', ['%type%' => $media->getBinaryContent()->getMimeType()])
```

Comment:
And this

Reply:
I pushed a commit that revert this indent.